src/library_manager/lib_manager.c

```diff
@@ -744,7 +836,7 @@ int lib_manager_load_library(uint32_t dma_id, uint32_t lib_id, uint32_t type)
 	/* allocate temporary manifest buffer */
 	man_tmp_buffer = (__sparse_force void __sparse_cache *)
 			rballoc_align(0, SOF_MEM_CAPS_DMA,
-				      MAN_MAX_SIZE_V1_8, dma_ext->addr_align);
+				      MAN_MAX_SIZE_V1_8, CONFIG_MM_DRV_PAGE_SIZE);
```

Comment:
how is this related? Can we add a comment in a follow-up? Or, if this wasn't intended, revert this

Reply:
Authentication engine in ROM_EXT requires page aligned buffer. Otherwise it fails.